src/application/application.cpp

```diff
@@ -96,11 +97,13 @@ void Application::setupAppMenu()
 void Application::on_open(const Application::type_vec_files& files,
                           __attribute__((unused)) const Glib::ustring& hint)
 {
+    std::vector<Glib::RefPtr<Gio::File>> files_to_open;
```

Comment:
The `files` parameter is actually a `std::vector<Glib::RefPtr<Gio::File>>`. You can use it as-is without copying its contents to a new vector.